pkg/operator/ceph/pool/controller.go

```diff
@@ -403,10 +404,20 @@ func configureRBDStats(clusterContext *clusterd.Context, clusterInfo *cephclient
 	return nil
 }
 
-func (r *ReconcileCephBlockPool) cancelMirrorMonitoring(cephBlockPoolName string) {
-	// Cancel the context to stop the go routine
-	r.blockPoolContexts[cephBlockPoolName].internalCancel()
```

Comment:
Although the diff looks like I deleted this :\, I extended this `cancelMonitoring` pattern to file and object controllers.